### PR TITLE
Version 0.7.13

### PR DESCRIPTION
**Pull request by jeffthieleke-wf**

### CODE REVIEW

File: js/tools/browserManager.js [hunk 1]
```diff
@@ -63,7 +63,9 @@ class BrowserManager { /* exported BrowserManager*/
     let dfTab = null;
     let openNewTab = this._alwaysOpenNewTab || openNewTabForce;
     let openNewTabForeground = openNewTabBackGroundForce ? false : this._openNewTabForeground;
-    let reuseDropFeedsTab = this._reuseDropFeedsTab;
+    let feedUrlStartPattern = 'blob:' + browser.extension.getURL('');
+    let isFeed = url.startsWith(feedUrlStartPattern);
```
Comment (jeffthieleke-wf) [inline js/tools/browserManager.js:67]: @dauphine-dev - we should probably rework BrowserManager.isDropFeedsTab() to work on URL strings instead duplicating the code here.

File: js/ui/sidebar/topMenu.js [hunk 2]
```diff
@@ -122,7 +122,8 @@ class TopMenu  { /*exported TopMenu*/
 
   static async updateAutomaticUpdateInterval() {
     let automaticUpdatesMinutes = await LocalStorageManager.getValue_async('automaticFeedUpdateMinutes', DefaultValues.automaticFeedUpdateMinutes);
-    let automaticUpdatesMilliseconds = Math.min(automaticUpdatesMinutes * 60000, 300000);
+    //let automaticUpdatesMilliseconds = Math.min(automaticUpdatesMinutes * 60000, 300000);
+    let automaticUpdatesMilliseconds = Math.max(automaticUpdatesMinutes * 60000, 300000);
```
Comment (jeffthieleke-wf) [inline js/ui/sidebar/topMenu.js:126]: @dauphine-dev - Good catch!

File: js/tools/browserManager.js (hunk 1 above)
Comment (dauphine-dev) [reply js/tools/browserManager.js:67]: yeah you are right

Comment (dauphine-dev) [reply js/tools/browserManager.js:67]: Done on 0.8 branch

**Pull request closed without merging**